README.md

```diff
@@ -67,9 +67,14 @@ config :sentry,
 | `release` | False  | None | |
 | `server_name` | False  | None | |
 | `use_error_logger` | False  | False | |
-| hackney_opts | False  | `[pool: :sentry_pool]` | |
-| hackney_pool_max_connections | False  | 50 | |
-| hackney_pool_timeout | False  | 5000 | |
+| `hackney_opts` | False  | `[pool: :sentry_pool]` | |
+| `hackney_pool_max_connections` | False  | 50 | |
+| `hackney_pool_timeout` | False  | 5000 | |
+| `enable_source_code_context` | True | | |
+| `root_source_code_path` | Required if `enable_source_code_context` is enabled | | Should generally be set to `File.cwd!`|
```

Comment:
Discussed offline, but for posterity, the issue is that it will get called from within the Sentry source code, not the project, causing it to only read source code within Sentry.